src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -155,7 +155,13 @@
 
     public @Option Miscellaneous MISCELLANEOUS = new Miscellaneous();
     public static class Miscellaneous {
-        public @Option @Num(max = 60 * 12, step = 10) int REFRESH_TIME = 60;
+        public static class RefreshTime {
+            public @Option @Num(max = 60 * 12, step = 10) int VALUE = 60;
+            public @Option boolean RANDOM = false;
+            public @Option @Num(min = 10) int RANDOM_MIN = 10;
+            public @Option @Num(min = 10, max = 120) int RANDOM_MAX = 60;
+        }
+        public @Option RefreshTime REFRESH_TIME = new RefreshTime();
```

Comment:
Just treat all of this as a range instead of 4 separate values. Make a custom editor that works like the current range with %, but for numbers. If you want randomness you say: min 60, max 70. if you don't want randomness, you say min 60 max 60.